dag/health.yml

```diff
@@ -30,6 +30,7 @@ steps:
     - data://garden/reference
   data://garden/who/2022-09-30/ghe:
     - data://meadow/who/2022-09-30/ghe
+    - data://garden/un/2022-07-11/un_wpp
```

Comment:
Do we need this? (it doesn't have any effect on the memory problem, just asking)

Reply:
Yeah, it is kind of hidden, but the function `add_population` uses this dataset.